recipe/patches/0020-Adjust-relative-path-for-libdevice.patch

```diff
@@ -0,0 +1,25 @@
+From d0f71b5998edbd36ca0911f797f5f1fa34a67910 Mon Sep 17 00:00:00 2001
+From: Mark Harfouche <mark.harfouche@gmail.com>
+Date: Sat, 27 Jan 2024 22:27:56 -0500
+Subject: [PATCH] Adjust relative path for libdevice
+
+---
+ .../third_party/tsl/tsl/platform/default/cuda_libdevice_path.cc | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/third_party/xla/third_party/tsl/tsl/platform/default/cuda_libdevice_path.cc b/third_party/xla/third_party/tsl/tsl/platform/default/cuda_libdevice_path.cc
+index ed2ffece..271f3141 100644
+--- a/third_party/xla/third_party/tsl/tsl/platform/default/cuda_libdevice_path.cc
++++ b/third_party/xla/third_party/tsl/tsl/platform/default/cuda_libdevice_path.cc
+@@ -51,7 +51,7 @@ std::vector<std::string> CandidateCudaRoots() {
+     // TF lib binaries are located in both the package's root dir and within a
+     // 'python' subdirectory (for pywrap libs). So we check two possible paths
+     // relative to the current binary for the wheel-based nvcc package.
+-    for (auto path : {"/../nvidia/cuda_nvcc", "/../../nvidia/cuda_nvcc"})
++    for (auto path : {"/../nvidia/cuda_nvcc", "/../../nvidia/cuda_nvcc", "/../../../.."})
```

Comment:
is there perhaps an easier way to just put `${PREFIX}` in here? 

cc: @jakirkham

Reply:
Is that a question for me? If so, I'm not really sure

Reply:
a little bit.

just pointing out a few gotchas with nvvm